journald/README.md

```diff
@@ -99,6 +99,29 @@ logs:
           - sshd.service
 ```
 
+##### Tailing the same journal multiple times
+
+In some situations it may be desirable to tail the same journal more than once in order to report different units with different source or service tags. 
```

Comment:
```suggestion
If you want to report units with different source or service tags, these must appear in separate journald configs.
``` 

Take it if you want it, to me the phrasing "in some situations it may be desirable" is a bit passive and hides the useful bit of this feature.